patches/third_party-blink-public-devtools_protocol-browser_protocol.pdl.patch

```diff
@@ -11,3 +11,26 @@ index 73feffdbb21001f6672d0987228018aaa3c4a0da..c41050934ffd322bcca8441b5ba17022
  
    # Reason for a permissions policy feature to be disabled.
    experimental type PermissionsPolicyBlockReason extends string
+@@ -7593,6 +7595,22 @@ domain Page
+   # Enables page domain notifications.
+   command enable
+ 
++  # Generates a Page Graph report for the page.
```

Comment:
can we patch in another file to generate_protocol_resources.py instead?

Reply:
this file is not used by `generate_protocol_resources.py`.
instead it's passed via gn to 3 scripts: `third_party/inspector_protocol/check_protocol_compatibility.py`, `third_party/inspector_protocol/concatenate_protocols.py` and `third_party/inspector_protocol/code_generator.py`.

I need to patch all 3 files or patch `third_party/inspector_protocol/pdl.py` on lower level. Not sure if it's worth it.

Reply:
can you compress this patch as much as possible (remove blank lines, comments, etc...)? Also I think adding it to the end of the file would reduce the chance of conflicts. Alternatively could we put these in a separate pdl file and do a patch for the `protocol_version` something like `inputs += brave_protocol_version_inputs deps += brave_protocol_version_deps args += brave_protocol_version_args`  where we run `check_protocol_compatibility.py` on our file (which goes into brave_protocol_version_deps)?

Reply:
> can you compress this patch as much as possible (remove blank lines, comments, etc...)?

I've implemented a python-based merge solution. No need to patch `browser_protocol.pdl` now at all.
https://github.com/brave/brave-core/pull/14697/commits/f00c1f27dac0be49166f244580304598ef461e63